TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/BNO055IMUUtil.java

```diff
@@ -50,4 +53,52 @@ public static void remapAxes(BNO055IMU imu, AxesOrder order, AxesSigns signs) {
             Thread.currentThread().interrupt();
         }
     }
+
+    /**
+     * Remaps the IMU coordinate system so that the remapped +Z faces the provided
+     * {@link AxisDirection}. See {@link #remapAxesIntuitive} for details about the remapping.
+     *
+     * @param imu IMU
+     * @param direction axis direction
+     */
+    public static void remapZAxis(BNO055IMU imu, AxisDirection direction) {
+        switch (direction) {
+            case POS_X:
+                remapAxesIntuitive(imu, AxesOrder.ZYX, AxesSigns.NPP);
```

Comment:
Should the first axis (i.e., the old X axis) be negated?

Based on the remap axis method comment, it's implied that this guarantees right-handedness. It seems easy enough to compute the parity of each component (each axis swap, each negation +1) and check that the frame is right handed before returning.

I haven't checked the following remaps.

Reply:
No, the first axis should not be negated. The signs correspond with the **new, remapped** axes.